initializers/dcim_interfaces.yml

```diff
@@ -0,0 +1,12 @@
+#- device: router1
+#  enabled: true
+#  form_factor: 0
```

Comment:
Would it be possible to use the actual word here, i.e. `Virtual`, `Other`, etc.?

See https://github.com/netbox-community/netbox-docker/blob/master/startup_scripts/110_devices.py#L52-L55 for how this was made for the `face` property in `devices.yml`.